### PR TITLE
catch null reference

### LINKED ISSUE #1598: Seafile catalog module causes load failure for catalog modules overview
At commit 

# Environment

 * Ampache version: 3e99df1
 * Web server + version: NginX 1.10.3
 * Server operating system: 32-bit Debian Buster (Testing)
 * Client operating system: 64-bit Debian Buster (Testing)

 * Client type:
   - Web

# Settings

Roughly standard ampache configuration. Nothing very different from stock.

# Description

Visit Modules -> Catalog Modules, content section is empty.

# Logs

Only entry in ampache log at level 5: 
```
2017-09-07 12:52:27 [roman] (autoload) -> 'Beets\Catalog' not found! 
```

Only entry in NginX error log: 

```
2017/09/07 12:54:01 [error] 18178#18178: *984 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Call to a member function get_format_string() on null in /srv/ampache/ampache-app/modules/catalog/seafile/seafile.catalog.php:484
Stack trace:
#0 /srv/ampache/ampache-app/templates/show_catalog_types.inc.php(42): Catalog_Seafile->format()
#1 /srv/ampache/ampache-app/admin/modules.php(199): require_once('/srv/ampache/am...')
#2 {main}
  thrown in /srv/ampache/ampache-app/modules/catalog/seafile/seafile.catalog.php on line 484" while reading response header from upstream, client: 10.0.0.16, server: vostro, request: "GET /admin/modules.php?action=show_catalog_types HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.0-fpm.sock:", host: "vostro:9988", referrer: "http://vostro:9988/index.php"
```

### PR DESCRIPTION
**Pull request by PlaidPhantom**
fixes #1598

### CODE REVIEW

**Comment by PlaidPhantom** [comment]
well this is embarrassing.

**Comment by wagnered** [comment]
@PlaidPhantom ,
Because the Travis build failed, I suspect you are not working with the latest commit (3e99df).  The latest php-cs-fixer is now included in the repository and dependent files have been modified to reflect the change.

**Comment by PlaidPhantom** [comment]
I'm not sure what's up. I was only behind one commit (the merge commit from my last PR, in fact). I rebased my commit on the latest `develop`, but that didn't fix the build. I definitely have [php-cs-fixer](https://github.com/PlaidPhantom/ampache/blob/develop/php-cs-fixer).

I did some testing locally; should [this line](https://github.com/ampache/ampache/blob/develop/scripts/tests/codestyle.sh#L6) be `elif hash ./php-cs-fixer` instead of `elif hash php-cs-fixer`?

**Comment by wagnered** [comment]
The Travis log indicates that php-cs-fixer or php-cs-fixer.phar does not exist in the root directory.

If you click on  "Details", it will take you to the build logs. click on one of the failed tests and it will tell  you what failed. As you created the PR, you should have access to the details.

**Comment by PlaidPhantom** [comment]
I can see the build logs. [php-cs-fixer](https://github.com/PlaidPhantom/ampache/blob/develop/php-cs-fixer) is _definitely_ in the repo root.

I think codestyle.sh might be looking for the global php-cs-fixer instead of the local one. On my local machine, `hash php-cs-fixer` looks for the command in $PATH, not locally.  To hash the local file, I have to run `hash ./php-cs-fixer`.  On the other hand, this still doesn't explain the failure because the build says that the `composer global require friendsofphp/php-cs-fixer:~2.4.0` command worked as well.

In other words, I'm not really sure what to try.

**Comment by wagnered** [comment]
@PlaidPhantom ,

>I can see the build logs. php-cs-fixer is definitely in the repo root.

Yes, if your reference to php-cs-fixer was located in "lib/vendor/friendsofphp/src/Fixer", I forgot to include a step to copy it to the root in composer.json. Try copying it to the root and running it from there.

If that folder doesn't exist, php-cs-fixer.phar can be downloaded into the root folder from [https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases](https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases).

>I think codestyle.sh might be looking for the global php-cs-fixer instead of the local one.

`scripts/tests/codestyle.sh` is what Travis runs during PRs. It expects php-cs-fixer.phar to be in the ampache (root) folder.

And it explains your following comment:

>On the other hand, this still doesn't explain the failure because the build says that the composer global require friendsofphp/php-cs-fixer:~2.4.0 command worked as well.

The Travis system cannot run your "Global" php-cs-fixer command.

I don't understand your use of the `hash` command: I have never used or needed to use it before

I hope this clears up the confusion.

**Comment by wagnered** [comment]
Oh yea, try running `php php-cs-fixer -v` in the root folder to see what version it actually is.

**Comment by wagnered** [comment]
@PlaidPhantom,

I will inquire at Travis-CI as to what the problem might be. I took the liberty of cloning your repository and running a few tests which were successful. I will probably merge your PR in a day or two.  Travis-CI is usually prompt at responding to queries.  Without this PR or at least the one commit, it is not possible to enable catalogs.

**Comment by PlaidPhantom** [comment]
Here's the output of those commands. Hope it helps!

```
$ php php-cs-fixer -v
PHP CS Fixer 2.4.0 Silver Gingerbread by Fabien Potencier and Dariusz Ruminski

Usage:
  command [options] [arguments]

Options:
  -h, --help            Display this help message
  -q, --quiet           Do not output any message
  -V, --version         Display this application version
      --ansi            Force ANSI output
      --no-ansi         Disable ANSI output
  -n, --no-interaction  Do not ask any interactive question
  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug

Available commands:
  describe     Describe rule / ruleset.
  fix          Fixes a directory or a file.
  help         Displays help for a command
  list         Lists commands
  readme       Generates the README content, based on the fix command help.
  self-update  [selfupdate] Update php-cs-fixer.phar to the latest stable version.

$ ./scripts/tests/codestyle.sh
./scripts/tests/codestyle.sh: line 6: hash: php-cs-fixer: not found
Please install or download latest stable php-cs-fixer
http://cs.sensiolabs.org/

```

**Comment by wagnered** [comment]
@PlaidPhantom, 

>./scripts/tests/codestyle.sh: line 6: hash: php-cs-fixer: not found

"$HOME/.composer/vendor/bin" must be in the $PATH for that command to work: It has to be added manually.  I don't think this is the problem of the travis-ci test failing, though.

codestyle.sh also checks for "php-cs-fixer.phar" in the root folder.

I will merge your PR and give more consideration about managing this problem.

**Comment by wagnered** [comment]
@PlaidPhantom 

Sorry, I sent the earlier comment to the wrong PR.

Support suggested that the changes made to their build system may be the cause and wanted us to try a modification to the .travis.yml file.

Could you save the following link as  [travis.patch](https://github.com/ampache/ampache/commit/5152609d1b721c0f79f149c376d080c106f95eb7.diff) the root folder , run it and push the change?
command: `patch .travis.yml travis.patch`

Thanks...

**Comment by wagnered** [comment]
Never mind! I needed the change to continue my development with a different catalog. 😄
